dev-tools/omdb/src/bin/omdb/db.rs

```diff
@@ -680,6 +685,53 @@ async fn cmd_db_sleds(
     Ok(())
 }
 
+#[derive(Tabled)]
+#[tabled(rename_all = "SCREAMING_SNAKE_CASE")]
+struct CustomerInstanceRow {
+    id: Uuid,
+    state: String,
+    propolis_id: Uuid,
+    sled_id: Uuid,
+}
+
+impl From<Instance> for CustomerInstanceRow {
+    fn from(i: Instance) -> Self {
```

Comment:
You don't need to, but it's pretty easy to add the serial number if you already
have the sled_id:

```
        let (_, my_sled) = LookupPath::new(opctx, datastore)                    
            .sled_id(my_sled_id)                                                
            .fetch()                                                            
            .await                                                              
            .context("failed to look up sled")?;                                
```
Then, the serial number is here:
```                                                                                
    my_sled.serial_number().to_string(), 
```

Reply:
Great idea! Since the id -> serial mapping is already visible today via `omdb db sleds`, I'll punt on this for now. I definitely want to make this command better (but want the basic data more now).